src/visual.ts

```diff
@@ -214,14 +216,24 @@ module powerbi.extensibility.visual {
                 if (_.isNumber(targetValue2)) {
                     toolTipItems.push({
                         value: targetValue2,
-                        metadata: categorical.TargetValue2 && categorical.TargetValue2[0]
+                        metadata: categorical.TargetValue2 && categorical.TargetValue2[0],
+                        customName: settings.tooltips.target2CustomName
                     });
                 }
 
-                let getRangeValue = (cValues: number[], sValue: number) => cValues ? cValues[idx] :
-                    (_.isNumber(targetValue) && _.isNumber(sValue) ? (sValue * targetValue / 100) : null);
+                let getRangeValue = (values: number[], percent: number, isMinimum?: boolean) => {
+                    let negativeMinimumCoef: number;
+                    if (isMinimum) {
+                        negativeMinimumCoef = values && values[idx] ? values[idx] : BulletChart.zeroValue;
+                    } else {
+                        negativeMinimumCoef = minimum;
+                    }
 
-                let minimum: number = getRangeValue(categoricalValues.Minimum, settings.values.minimumPercent);
+                    return values ? values[idx] :
+                        (_.isNumber(targetValue) && _.isNumber(percent) ? (percent * (targetValue - negativeMinimumCoef) / 100) + negativeMinimumCoef : null);
```

Comment:
Do we need to use lodash here? Can we use `isNaN` or `isFinite` or `value !==undefined && value !== null` instead?